tasks/pages/ward/[uuid].tsx

```diff
@@ -0,0 +1,120 @@
+import { useState } from 'react'
+import type { NextPage } from 'next'
+import Head from 'next/head'
+import { tw } from '@helpwave/common/twind/index'
+import { useAuth } from '../../hooks/useAuth'
+import { useRouter } from 'next/router'
+import { Header } from '../../components/Header'
+import { UserMenu } from '../../components/UserMenu'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import { TwoColumn } from '../../components/layout/TwoColumn'
+import type { BedDTO, RoomDTO } from '../../mutations/room_mutations'
+import {
+  useRoomQuery,
+  useCreateMutation,
+  useDischargeMutation,
+  useUpdateMutation
+} from '../../mutations/room_mutations'
+import { RoomOverview } from '../../components/RoomOverview'
+import { PatientDetail } from '../../components/layout/PatientDetails'
+
+type BedsPageTranslation = {
+  beds: string,
+  roomOverview: string
+}
+
+const defaultBedsPageTranslation = {
+  en: {
+    beds: 'Betten',
+    roomOverview: 'Room Overview'
+  },
+  de: {
+    beds: 'Bett',
+    roomOverview: 'Raum Übersicht'
```

Comment:
`Stationsliste` bzw. `Ward list`

Reply:
that is confusing, under the name "Stationsliste" I would imagine a list of all Stations